lago/prefix.py

```diff
@@ -1220,16 +1221,35 @@ def build(self, conf):
 
         utils.invoke_in_parallel(build.Build.build, builders)
 
+    @sdk_utils.expose
     def export_vms(
         self,
-        vms_names,
-        standalone,
-        export_dir,
-        compress,
-        init_file_name,
-        out_format,
+        vms_names=None,
+        standalone=False,
+        export_dir='.',
+        compress=False,
+        init_file_name='LagoInitFile',
+        out_format=YAMLOutFormatPlugin(),
         collect_only=False
     ):
+        """
+        Export vm images disks and init file.
+        The exported images and init file can be used to recreate
+        the environment.
+
+        Args:
+            vm_names(list of str): Names of the vms to export, if None
+                export all the vms in the env
+            standalone(bool): If false, export a layered image
+            export_dir(str): Dir to place the exported images and init file
+            compress(bool): If True compress the images with xz
+            init_file_name(str): The name of the exported init file
+            out_format(lago.plugins.output.OutFormatPlugin):
+                The type of the exported init file (the default is yaml)
+            collect_only(bool): If true, return only a mapping from vm name
+                to the disks that will be exported.
+
```

Comment:
(again github messed the comments so I can't see the change) - Thinking about it - it might be useful to always return some metadata of the export, including some finalized data(such as where is the .tar.gz):
For example, (semi-pesudoish here):
```python
     export_results = sdk.export(....)
     compressed_export = export_results.get('compressed_file')
     os.cp(compressed_export, os.path.join('exported-artifacts', 'export'))
```